example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -367,6 +367,55 @@ describe('The Sync Granules workflow', () => {
     });
   });
 
+  describe('when an ACL is provided in the task config and set to disabled', () => {
+    const meta = { ACL: 'disabled' };
+    let executionArn;
+    let lambdaOutput;
+    let objectACLs;
+    let files;
+    let key1;
+    let key2;
+
+    beforeAll(async () => {
+      const execution = await buildAndExecuteWorkflow(
+        config.stackName, config.bucket, workflowName, collection, provider, inputPayload, meta
+      );
+
+      executionArn = execution.executionArn;
+      console.log(`Wait for completed execution ${executionArn}`);
+
+      await waitForCompletedExecution(executionArn);
+
+      lambdaOutput = await lambdaStep.getStepOutput(executionArn, 'SyncGranule');
+      files = lambdaOutput.payload.granules[0].files;
+      key1 = s3Join(files[0].key);
+      key2 = s3Join(files[1].key);
+
+      await Promise.all([
+        s3ObjectExists({ Bucket: files[0].bucket, Key: key1 }),
+        s3ObjectExists({ Bucket: files[1].bucket, Key: key2 }),
+      ]);
+      objectACLs = await Promise.all(files.map(
+        (file) => s3().getObjectAcl({
+          Bucket: file.bucket,
+          Key: file.key,
+        })
+      ));
+    });
+
+    afterAll(async () => {
+      await deleteExecution({ prefix: config.stackName, executionArn: executionArn });
```

Comment:
I _think_ we also need to do a couple `deleteS3Object`s here? I haven't actually written a new test that creates S3 objects so I could be wrong and it's not necessary.

Reply:
Yeah, I thought that as well! But I _think_ the afterAll in the beginning of the spec deletes the entire folder/path in S3 which handles deleting the objects with it https://github.com/nasa/cumulus/blob/master/example/spec/parallel/syncGranule/SyncGranuleSuccessSpec.js#L178

Reply:
I added the delete calls anyway 😄  [`d01be12` (#2946)](https://github.com/nasa/cumulus/pull/2946/commits/d01be12a10703b75204c5f22ea23328cbacb13aa#diff-6b021707aab039821866154177097c5c7c8a23911b14bf7012dc6437528e5882R409-R410)

Reply:
Oh cool! I didn't see the `deleteFolder` call. Yeah, I would also suspect that that would do it. 

Probably doesn't hurt to have redundant deletes here. The downside I guess would be if the `deleteFolder` does clean up S3 and then the new delete calls clutter up the logs with errors. Not sure how much I care about that 😄 . Definitely not worth holding anything up.